fs/fs.go

```diff
@@ -108,14 +123,17 @@ func (fs *Fs) RecordFilepaths(dir string, less func(a, b string) bool) ([]string
 		if item.IsDir() {
 			continue
 		}
+
 		filepaths = append(filepaths, filepath.Join(dir, item.Name()))
 	}
 
-	sort.Slice(filepaths, func(i, j int) bool {
-		return less(filepaths[i], filepaths[j])
+	filteredFilepaths := fs.filterBackupFiles(filepaths)
```

Comment:
Same question here

Reply:
As mentioned, I updated the code to remove the helper function and check for backup files in the preceding for loop.